Library/Formula/sqlite.rb

```diff
@@ -7,6 +7,7 @@ class Sqlite < Formula
 
   bottle do
     cellar :any
+    revision 1
```

Comment:
Wrong place, here. Needs to be underneath the `sha256` in the _initial_ block, not inside the bottle block.